noir-projects/aztec-nr/aztec/src/note/lifecycle.nr

```diff
@@ -18,7 +17,7 @@ pub fn create_note<Note, N, M>(
     let header = NoteHeader { contract_address, storage_slot, nonce: 0, note_hash_counter };
     // TODO: change this to note.set_header(header) once https://github.com/noir-lang/noir/issues/4095 is fixed
     Note::set_header(note, header);
-    let inner_note_hash = compute_note_hash_for_insertion(*note);
+    let inner_note_hash = compute_inner_note_hash(*note);
```

Comment:
`compute_note_hash_for_insertion` was just a wrapper around `compute_inner_note_hash` which was just making the code messier. I am fine with having a different name than inner note hash but if we introduce it should be consistent across protocol circuits and aztec.nr. It was not the case at this point.